bundles/org.openhab.binding.myuplink/src/main/resources/OH-INF/i18n/myuplink.properties

```diff
@@ -0,0 +1,3 @@
+# FIXME: please add all English translations to this file so the texts can be translated using Crowdin

```

Comment:
Please run i18n tool when the Thing structure is stable and also add the i18n keys.